lite2/client_benchmark_test.go

```diff
@@ -20,12 +20,12 @@ import (
 // or -benchtime 100x.
 //
 // Remember that none of these benchmarks account for network latency.
-var (
-	benchmarkFullNode = mockp.New(GenMockNode(chainID, 1000, 100, 1, bTime))
-	genesisHeader, _  = benchmarkFullNode.SignedHeader(1)
-)
```

Comment:
Made these local inside benchmark functions as they take a long time to create and are not required for tests.